### PR TITLE
chore(filesystem): improve docblock descriptions

### PR DESCRIPTION
**Pull request by jrmajor**
Some of these changes are fixes of obvious mistakes, but the rest may be a bit opinionated. I'm open to discussing/reverting some of them :)

### CODE REVIEW

File: src/Psl/Filesystem/get_modification_time.php [hunk 1]
```diff
@@ -16,6 +16,8 @@
  *
  * @throws Exception\NotFoundException If $node is not found.
  * @throws Exception\RuntimeException In case of an error.
+ *
+ * @return int The last content modification time as a Unix timestamp.
```
Comment (azjezz) [inline src/Psl/Filesystem/get_modification_time.php:20]:
```suggestion
 * @return int<0, max> The last content modification time as a Unix timestamp.
```

**Review by azjezz — COMMENTED** [review]
Thank you :)

one minor change.

File: src/Psl/Filesystem/get_modification_time.php (hunk 1 above)
Comment (jrmajor) [reply src/Psl/Filesystem/get_modification_time.php:20]:
```sh
touch -d "1960-05-19T12:00:00" test_file
```

```php
Psl\Filesystem\get_modification_time('test_file') === -303573600
```

**Comment by jrmajor** [comment]
All `@param` annotations I checked are full sentences (begin with an uppercase letter and end with a period), but there are inconsistencies with `@return`. Do you have any preference on how they should be standardized?

**Comment by azjezz** [comment]
> preference on how they should be standardized?

no, as long as things are consistent :)

File: src/Psl/Filesystem/get_modification_time.php (hunk 1 above)
Comment (azjezz) [reply src/Psl/Filesystem/get_modification_time.php:20]: ah, ignore me then :p

**Comment by jrmajor** [comment]
Let's go with full sentences everywhere then :)

**Review by azjezz — APPROVED** [review]

**Pull request merged**